dynamicforms/mixins/allow_tags.py

```diff
@@ -46,6 +46,9 @@ def to_internal_value(self, data):
         Validator is taken directly from DRF and updated, to check for allow_tags.
         Option allow_tags enables adding value that is not on the options list.
         """
+        if not self.allow_tags:
```

Comment:
Tole je zihr bug: allow tags setting omogoča custom vnos vrednosti v select. Ti pa spremeniš delovanje komponente, da v takih primerih gre po bližnjici. To sva že debatirala in sem ti že enkrat zavrnil. Kar težko verjamnem, da gredo unit testi skoz. Verjetno zato, ker nimamo ustreznih unit testov za testiranje multi select choice fielda. Predlagam, da odpreš nalogo, ki ga bo unit testirala (selenium, vključno s testom, kjer je allow_tags=True)

Unit testi:
- nekje narediš multi-select choices (Validated.item_flags je bil vedno mišljen kot multiselect choice - validator je že celo kul)
- narediš unit test, ki testira, da lahko nabereš eno, dve, tri opcije.
- narediš tudi unit test, ki poskuša "izbrati" "D" opcijo (in mora seveda odleteti, ker je validator ne dovoli)
- popraviš validator, da dovoli tudi choice "E", vklopiš allow_tags in stestiraš tudi, če gre dodajanje vrednosti "E". In če faila pri vrednosti "F"